pytorch_impl/datareader.py

```
@@ -0,0 +1,403 @@
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
+
+
+def pprint(text: str) -> None:
+    print(f"{bcolors.OKGREEN}{text}{bcolors.ENDC}")
+
+
+class DataReader:
```

Pylint (3.3.7)

[refactor] Too many instance attributes (30/7)

(R0902)